src/test/mediapipeflow_test.cpp

```diff
@@ -614,10 +718,65 @@ TEST_F(MediapipeConfigChanges, AddImroperGraphThenFixWithReloadThenBreakAgain) {
     modelManager.loadConfig(configFilePath);
     definition = factory.findDefinitionByName(mgdName);
     ASSERT_NE(nullptr, definition);
-    ASSERT_EQ(definition->getStatus().getStateCode(), PipelineDefinitionStateCode::LOADING_PRECONDITION_FAILED);
+    ASSERT_EQ(definition->getStatus().getStateCode(), PipelineDefinitionStateCode::RETIRED);
```

Comment:
Why this change is required?

Reply:
Here the test should show that changing back to improper config will cause the reload and the mediapipe will be in failed load state.